app/src/main/java/org/fossasia/phimpme/leafpic/activities/SingleMediaActivity.java

```diff
@@ -594,6 +627,31 @@ public void folderSelected(String path) {
         return super.onOptionsItemSelected(item);
     }
 
+    public EditText getDescriptionDialog(final ThemedActivity activity, AlertDialog.Builder descriptionDialog){
+
+        final View DescriptiondDialogLayout = activity.getLayoutInflater().inflate(R.layout.dialog_description, null);
+        final TextView DescriptionDialogTitle = (TextView) DescriptiondDialogLayout.findViewById(R.id.description_dialog_title);
+        final CardView DescriptionDialogCard = (CardView) DescriptiondDialogLayout.findViewById(R.id.description_dialog_card);
+        final EditText editxtDescription = (EditText) DescriptiondDialogLayout.findViewById(R.id.description_edittxt);
+
+        DescriptionDialogTitle.setBackgroundColor(activity.getPrimaryColor());
+        DescriptionDialogCard.setBackgroundColor(activity.getCardBackgroundColor());
+        ThemeHelper.setCursorDrawableColor(editxtDescription, activity.getTextColor());
+        editxtDescription.getBackground().mutate().setColorFilter(activity.getTextColor(), PorterDuff.Mode.SRC_ATOP);
+        editxtDescription.setTextColor(activity.getTextColor());
+
+        realm = Realm.getDefaultInstance();
+        databaseHelper =new DatabaseHelper(realm);
+        temp= databaseHelper.getImageDesc(pathForDescription);
+        if(temp != null) {
+            editxtDescription.setText(temp.getTitle());
+        }
+        else
+            editxtDescription.setHint(getString(R.string.description_hint));
```

Comment:
is `hint` causing the cursor to not in center
![image](https://user-images.githubusercontent.com/6936968/27983688-3ba70748-63e0-11e7-8670-04943673cbcd.png)